habitat_baselines/common/utils.py

```diff
@@ -53,6 +56,51 @@ def forward(self, x):
         return CustomFixedCategorical(logits=x)
 
 
+class ResizeCenterCropper(nn.Module):
+    def __init__(
+        self, force_input_size: Optional[tuple], channels_first: bool = True
```

Comment:
`force_input_size` name is a little bit confusing. Is that input size for transformation or size of input to the next level. Maybe, use naming similar to [torchvision.transforms.CenterCrop]:(https://pytorch.org/docs/stable/torchvision/transforms.html#torchvision.transforms.CenterCrop)
`size (sequence or python:int) – Desired output size of the crop. If size is an int instead of sequence like (h, w), a square crop (size, size) is made.`

Regarding `channels_first` maybe call it like `NCHW` and add docstring. Because, channels are not really first and to be more explicit.

Reply:
It also resizes the input the though, it doesn't just center crop. I also kept it like so because at least for center cropping I wrote the code such that ....CHW or ...HWC (regardless of the number of channels) were supported. The channels_first is a convention from Tensorflow/Keras.

Reply:
I changed to channels_list since that is true regardless of the channels number.